onnxmltools/convert/common/_topology.py

```diff
@@ -243,6 +243,8 @@ def __init__(self, model, default_batch_size=1, initial_types=None,
         name and a type defined in data_types.py.
         :param reserved_variable_names: A set of strings which are not allowed to be used as a variable name
         :param reserved_operator_names: A set of strings which are not allowed to be used as a operator name
+        :param custom_conversion_functions: a dictionary for specifying the user customized conversion function
```

Comment:
`user customized`--->`customized`. The word `user` looks redundant.

Reply:
either 'user customized' or 'custom'